cmd/edit/machinepool/cmd.go

```diff
@@ -52,7 +53,7 @@ func NewEditMachinePoolCommand() *cobra.Command {
 		Long:    long,
 		Aliases: aliases,
 		Example: example,
-		Args:    cobra.MinimumNArgs(1),
+		Args:    upgrade.NewUpgradeArgsFunction(false),
```

Comment:
as it is used also in a non upgrade scenario, to avoid confusion I would suggest to rename it to `NewMachinepoolArgsFunction`and move it somewhere under pkg/machinepool

Reply:
Good point. I renamed the function and moved it under `pkg/machinepool`.